### DISCUSSION TITLE
Welcome to rhino-editor Discussions!

### DISCUSSION THREAD

**Discussion by KonnorRogers** [KonnorRogers/rhino-editor#152] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brendon** [comment]
Hi @KonnorRogers, I just wanted to say thank you for starting this project! One of the biggest source of complaints from our customers is around the limitations of the trix editor. The way it doesn't support styles or classes on block elements makes a lot of things impossible. TipTap is moving along really well with features and this project makes the bridge to there from trix much more straightforward.

Have you thought about letting Ruby Weekly know about this project? Also, if you're comfortable with it, you should set up a funding file so you can accept sponsorship for your work :)

**Reply by KonnorRogers** [reply]
@brendon Doh! totally forgot to add the funding.yml! Thanks!

I have not thought of reaching out to Ruby Weekly. To me it seems like a Ruby only newsletter, which this is not really Ruby 😆

**Reply by brendon** [reply]
Haha! Hopefully you get a bit of sponsorship now! :D

Ruby Weekly features a lot of Rails stuff so your package would definitely fit in. I'm currently trying to get them to mention my new list management gem (https://github.com/brendon/positioning) but so far no dice.
